SoftLayer/CLI/autoscale/create.py

```diff
@@ -0,0 +1,137 @@
+        {
+            "amount": args['policy_amount'],
+            "scaleType": args['policy_relative']
+        }
+    ]
+    policy_template = {
+        'name': args['policy_name'],
+        'policies': scale_actions
+
+    }
+    policies = []
+
+    block = []
+    number_disk = 0
+    for guest_disk in args['disk']:
+        disks = {'diskImage': {'capacity': guest_disk}, 'device': number_disk}
+        block.append(disks)
+        number_disk += 1
+
+    virt_template = {
+        'localDiskFlag': False,
+        'domain': args['domain'],
+        'hostname': args['hostname'],
+        'sshKeys': ssh_keys,
+        'postInstallScriptUri': args.get('postinstall'),
+        'operatingSystemReferenceCode': args['os'],
+        'maxMemory': args.get('memory'),
+        'datacenter': {'id': datacenter[0]['id']},
+        'startCpus': args.get('cpu'),
+        'blockDevices': block,
+        'hourlyBillingFlag': True,
+        'privateNetworkOnlyFlag': False,
+        'networkComponents': [{'maxSpeed': 100}],
+        'typeId': 1,
+        'userData': [{
+            'value': args.get('userdata')
+        }],
+        'networkVlans': [],
+
+    }
+
+    order = {
+        'name': args['name'],
+        'cooldown': args['cooldown'],
+        'maximumMemberCount': args['maximum'],
+        'minimumMemberCount': args['minimum'],
+        'regionalGroupId': args['regional'],
+        'suspendedFlag': False,
+        'balancedTerminationFlag': False,
+        'virtualGuestMemberTemplate': virt_template,
+        'virtualGuestMemberCount': 0,
+        'policies': policies.append(clean_dict(policy_template)),
+        'terminationPolicyId': args['termination_policy']
+    }
+
+    # print(virt_template)
+
+    for pod in pods:
+        if args.get('datacenter') in str(pod['name']):
+            closure.append(pod['name'])
+    click.secho(click.style('Warning: Closed soon: %s' % (', '.join(closure)), fg='yellow'))
+    if not (env.skip_confirmations or formatting.confirm(
+            "This action will incur charges on your account. Continue?")):
+        raise exceptions.CLIAbort('Aborting scale group order.')
+    else:
+        result = scale.create(order)
+
+        table = formatting.KeyValueTable(['name', 'value'])
+        table.align['name'] = 'r'
+        table.align['value'] = 'l'
+        table.add_row(['Id', result['id']])
+        table.add_row(['Created', result['createDate']])
+        table.add_row(['Name', result['name']])
+        table.add_row(['Virtual Guest Id', result['virtualGuestMembers'][0]['virtualGuest']['id']])
+        table.add_row(['Virtual Guest domain', result['virtualGuestMembers'][0]['virtualGuest']['domain']])
+        table.add_row(['Virtual Guest hostname', result['virtualGuestMembers'][0]['virtualGuest']['hostname']])
+        output = table
+
+    env.fout(output)
+
+
+def clean_dict(dictionary):
+    """Removes any `None` entires from the dictionary"""
+    return {k: v for k, v in dictionary.items() if v}
```

Comment:
This would actually make a good addition to the `utils` package, I know of a few other areas in the code it might be useful. So lets move it to `SoftLayer.utils`, thanks